### DISCUSSION TITLE
Can't resolve '@ark-ui/react/accordion'

### DISCUSSION THREAD

**Discussion by karolinelemos** [chakra-ui/chakra-ui#9205] [Q&A]
Hi! I'm updating the chakra to use v3, but this error is happening:

![error](https://github.com/user-attachments/assets/3240fde0-a87c-4963-b79a-779ea40a9df8)

If I update the in the file to use `@ark-ui/react` instead of `@ark-ui/react/accordion` the error appears in other files.

Any idea to resolve?

Thank u

**Comment by segunadebayo** [comment]
Please check your tsconfig to ensure it matches what we have in the docs
https://www.chakra-ui.com/docs/get-started/installation#update-tsconfig

Also check your node version to be sure you’re on >= 20

**Reply by karolinelemos** [reply]
Hi! I don't use TypeScript and my node version is `20.17.0`

**Reply by karolinelemos** [reply]
I updated webpack and it worked! Thanks for the support :rocket:
